invertedai/cosimulation.py

```diff
@@ -33,30 +33,45 @@ def __init__(
         location: str,
         ego_agent_mask: Optional[List[bool]] = None,
         monitor_infractions: bool = False,
-        render_birdview: bool = False,
+        get_birdview: bool = False,
         # sufficient arguments to initialize must also be included
         **kwargs,
     ):
         self._location = location
-        response = initialize(location=location, **kwargs)
+        response = initialize(
+            location=location,
+            get_birdview=get_birdview,
+            get_infractions=monitor_infractions,
+            **kwargs,
+        )
+        if monitor_infractions and (response.infractions is not None):
+            # TODO: package infractions in dataclass
+            self._infractions = (
+                [inf.collisions for inf in response.infractions],
+                [inf.offroad for inf in response.infractions],
+                [inf.wrong_way for inf in response.infractions],
```

Comment:
Does this still make sense? I think we have the required dataclass now, which we're unnecessarily deconstructing here.